### PR TITLE
vendor github.com/containers/image/v5@v5.5.0

### PR DESCRIPTION
**Pull request by vrothberg**
Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

File: vendor/github.com/containers/image/v5/version/version.go [hunk 1]
```diff
@@ -6,12 +6,12 @@ const (
...
 	// VersionMinor is for functionality in a backwards-compatible manner
-	VersionMinor = 4
+	VersionMinor = 5
 	// VersionPatch is for backwards-compatible bug fixes
-	VersionPatch = 4
+	VersionPatch = 1
 
 	// VersionDev indicates development branch. Releases will be empty string.
-	VersionDev = ""
+	VersionDev = "-dev"
```
Comment (mtrmac) [inline vendor/github.com/containers/image/v5/version/version.go:14]: This is not ideal.

**Comment by vrothberg** [comment]
@mtrmac updated. Let's see how the goproxy deals with a retag.

**Comment by mtrmac** [comment]
https://travis-ci.org/github/containers/skopeo/jobs/699336876 . (No idea why GitHub ins’t linking to it.)

I suppose we could overwrite `GOPROXY` in `Makefile`, and revert later… ugly. It’s almost tempting to just point the tag back — almost.

**Comment by vrothberg** [comment]
Need to cut a v5.5.1 to resolve the GOPROXY issues...

**Pull request closed without merging**